nip47/notifications/nip47_notifier_test.go

```diff
@@ -82,10 +83,17 @@ func doTestSendNotificationPaymentReceived(t *testing.T, svc *tests.TestService,
 	notifier := NewNip47Notifier(relay, svc.DB, svc.Cfg, svc.Keys, permissionsSvc, transactionsSvc, svc.LNClient)
 	notifier.ConsumeEvent(ctx, receivedEvent)
 
-	assert.NotNil(t, relay.PublishedEvent)
-	assert.NotEmpty(t, relay.PublishedEvent.Content)
+	var publishedEvent *nostr.Event
+	if cipher.Version == "0.0" {
```

Comment:
Is this okay? cc @rolznz

Doesn't feel good to me

Reply:
Yeah, we should try avoid `if`s in the test as much as possible. You could pass the index as an argument to this method, or the event kind, and get the event from the array matching the kind